### PR TITLE
Fulcrum: add an ability to browse under someone's account

### PR DESCRIPTION
**Pull request by Kharabet**

### CODE REVIEW

File: packages/fulcrum/src/layout/FooterVersion.tsx [hunk 1]
```diff
@@ -1,15 +1,24 @@
-import React, { Component } from 'react'
+import React from 'react'
+import ImepsonateInput from '../components/ImpersonateImput'
```
Comment (AoDev) [inline packages/fulcrum/src/layout/FooterVersion.tsx:2]: Typo

```suggestion
import ImpersonateInput from '../components/ImpersonateInput'
```

File: packages/fulcrum/src/services/FulcrumProvider.ts [hunk 2]
```diff
@@ -71,7 +71,7 @@ const initialNetworkId = getNetworkIdByString(networkName)
 
 export class FulcrumProvider {
   public static Instance: FulcrumProvider
-
+  public impersionateAddress = ''
```
Comment (AoDev) [inline packages/fulcrum/src/services/FulcrumProvider.ts:74]:
```suggestion
  public impersonateAddress = ''
```

File: packages/fulcrum/src/services/FulcrumProvider.ts [hunk 3]
```diff
@@ -162,6 +162,13 @@ export class FulcrumProvider {
     return FulcrumProvider.Instance
   }
 
+  private getCurrentAccount(): string | undefined {
+    return this.impersionateAddress
```
Comment (AoDev) [inline packages/fulcrum/src/services/FulcrumProvider.ts:166]:
```suggestion
    return this.impersonateAddress
```

Nice refactor :)

File: packages/fulcrum/src/components/ImpersonateImput.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react'
+import { FulcrumProvider } from '../services/FulcrumProvider'
+import Web3Utils from 'web3-utils'
+import { FulcrumProviderEvents } from '../services/events/FulcrumProviderEvents'
+import { ProviderChangedEvent } from '../services/events/ProviderChangedEvent'
+
+export function ImepsonateInput() {
```
Comment (AoDev) [inline packages/fulcrum/src/components/ImpersonateImput.tsx:7]: typo
```suggestion
export function ImpersonateInput() {
```

**Review by AoDev — CHANGES_REQUESTED** [review]
Just saw some typos, but looks good otherwise. Nice refactor  : )

**Comment by Kharabet** [comment]
Oh... How come I made a typo...

**Review by AoDev — APPROVED** [review]
Cool 👍

**Pull request merged**